test/tests/functional/pbs_qmgr.py

```diff
@@ -209,141 +160,18 @@ def test_multi_attributes(self):
             self.assertTrue(msg in e.msg[0])
         self.server.expect(NODE, 'queue', op=UNSET, id=self.mom.shortname)
 
-    @tags('multisched')
-    def test_list_sched(self):
+    def test_list_default_sched(self):
```

Comment:
This test duplicates the test you put just moved into the multisched test plan.  There is no need for it here.

Reply:
This is for the scenario 1: Only default sched exist. Since there is no multiple scheds, I feel its appropriate here. Please let me know If you still want to remove this test case.

Reply:
I still think this test is a repeat of what is in the multisched test plan.  It isn't needed here.

Reply:
OK deleted this test case.